cmd/cstor-pool-mgmt/pool/v1alpha2/import.go

```diff
@@ -0,0 +1,54 @@
+package v1alpha2
+
+import (
+	"github.com/golang/glog"
+	"github.com/openebs/maya/cmd/cstor-pool-mgmt/controller/common"
+	"github.com/openebs/maya/cmd/cstor-pool-mgmt/volumereplica"
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	api "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha2"
+	zfs "github.com/openebs/maya/pkg/zfs/cmd/v1alpha1"
+)
+
+// Import will import pool for given CSP object.
+// It will also set `cachefile` property for that pool
+// if it is mentioned in object
+func Import(csp *api.CStorNPool) (string, bool, error) {
```

Comment:
this code need to be revisited to remove the (possible) unnecessary code like GetVolumes..
having too many return values, which is not good, can be due to this..
can you add comments about the return values

Reply:
ok